source/common/upstream/eds.cc

```diff
@@ -110,9 +114,9 @@ void EdsClusterImpl::onConfigUpdate(const ResourceVector& resources) {
   onPreInitComplete();
 }
 
-bool EdsClusterImpl::updateHostsPerLocality(uint32_t priority, HostSet& host_set,
-                                            const HostVector& new_hosts,
-                                            LocalityWeightsMap& locality_weights_map) {
+bool EdsClusterImpl::updateHostsPerLocality(HostSet& host_set, const HostVector& new_hosts,
```

Comment:
Sorry, one more request - can we have consistent old -> new ordering or new->old ordering?
hosts / new_hosts
new_weights / weights could cause confusion

LGTM other than that